Get-help.md

```diff
@@ -0,0 +1,139 @@
+
+You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
+
+## How to Ask Good Questions
+
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup related and General questions. You can start following the sections below to understand how you can ask each of them.
+
+### Setup related questions
+
+**Helpful advice**: You can choose any OS you’re comfortable with. However, most contributors use Linux or MacOS. Getting Oppia up and running on Windows can be a challenge.
+
+#### Before you ask a question
+
+1. You can setup/install Oppia by visiting [[this|Installing-Oppia]] page.  Make sure you follow **all** the mentioned instructions from the beginning in a step by step manner.
+
+2. Make sure each step succeeds (verify it with the expected behavior if mentioned in wiki).
+
+3. In case of any unexpected behavior/errors at any step, make sure you check out our wiki on [how to troubleshoot when you are facing installation errors](https://github.com/oppia/oppia/wiki/Issues-with-installation%3F#when-you-encounter-an-installation-error).
+
+If you are still not able to fix your error, start following the section below to raise your question on [Gitter](https://gitter.im/oppia/oppia-chat#).
+
+#### How to ask a setup question
+
+**Note**: If you are stuck at Step X, we will assume all previous steps through X-1 were successful for you. In case there were any previously failed steps, kindly mention those too with their error logs.
+
+Please follow the template given below (mark x inside checkboxes to tick them) for sending your question to Gitter.
+
+```md
+**Checklist**
+- [ ]  I have followed the “Before you ask a question” section
+
+**System Information**
+- OS: (Be specific Ex: Ubuntu 20.04 or Ubuntu 20.04 VM on MacOS 11.2.1)
+
+**Error log**
+
+//paste error log here
+
+or
+
+paste a screenshot
+
+**Approaches already used to resolve the issue**
+
+(eg: Link to a stackoverflow answer or any solution that you have tried)
+- enter any additional description
+```
+
+## General questions
+
+### Before you ask a question
+
+1. We expect that you have already **set up Oppia on your machine**, and it is successfully running. (If not, kindly do that first!)
+2. Prepare a debugging doc following [[the guidelines provided on the wiki page|Debugging-Docs]].
+3. If there are **failing e2e tests** on your PR, and you haven’t done any changes in that direction, kindly understand that sometimes they just fail due to flakiness. You should request for a re-run of those only when it’s preventing your PR from getting merged.
```

Comment:
Maybe link to the wiki page describing "flakiness"?

Reply:
done